src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -642,9 +642,10 @@ private static ulong CurlReceiveHeadersCallback(IntPtr buffer, ulong size, ulong
                             response.Content.Headers.Clear();
 
                             easy._isRedirect = easy._handler.AutomaticRedirection &&
-                                         (response.StatusCode == HttpStatusCode.Redirect ||
-                                         response.StatusCode == HttpStatusCode.RedirectKeepVerb ||
-                                         response.StatusCode == HttpStatusCode.RedirectMethod);
+                                (response.StatusCode == HttpStatusCode.Moved ||           // 301
```

Comment:
Redirects are a test gap in general... the only redirect tests currently are for 302, with none for 303, 307, or 301.  And I don't have access to update the corefx-net.cloudapp.net site and the Redirect.ashx endpoint.